nano/node/cli.cpp

```diff
@@ -609,17 +608,22 @@ std::error_code nano::handle_node_options (boost::program_options::variables_map
 						ec = nano::error_cli::generic;
 					}
 				}
+				else if (account_str == "all")
+				{
+					auto transaction (node.node->store.tx_begin_write ());
+					reset_confirmation_heights (transaction, node.node->network_params.ledger, node.node->store);
+					std::cout << "Confirmation heights of all accounts (except genesis which is set to 1) are set to 0" << std::endl;
+				}
 				else
 				{
-					std::cerr << "Invalid account id\n";
+					std::cerr << "Specify either valid account id or 'all'\n";
 					ec = nano::error_cli::invalid_arguments;
 				}
 			}
 			else
 			{
-				auto transaction (node.node->store.tx_begin_write ());
-				reset_confirmation_heights (transaction, node.node->network_params.ledger, node.node->store);
-				std::cout << "Confirmation heights of all accounts (except genesis which is set to 1) are set to 0" << std::endl;
+				std::cerr << "confirmation_height_clear command requires one <account> option\n";
```

Comment:
It could also say: "or 'any' to clear all accounts.

Reply:
Improved this and also the help message to the `--confirmation_height_clear` option.